### PR TITLE
[EWL-5267] SG2 | Create "Category Page Hero" molecule

### PR DESCRIPTION
**Pull request by csj1012**
## Ticket(s)

**Jira Ticket**
- [EWL-5267: SG2 | Create "Category Page Hero" molecule](https://issues.ama-assn.org/browse/EWL-5267)

## Description
Add the Category Hero molecule to SG2.

## To Test
- [ ] `gulp serve`
- [ ] Visit Molecules > Category Hero
- [ ] Given that I am viewing SG2
- [ ] When I click "Molecules"
- [ ] Then I should see "Category Page Hero"
- [ ] When I click "Category Page Hero"
- [ ] Then I should see a large image
- [ ] And I should see a heading
- [ ] And the heading should be myriad pro
- [ ] And the heading should be semibold 38/44.2
- [ ] Given that I resize the browser to mobile
- [ ] Then I should see the image
- [ ] And I should see the purple eyebrow in all caps above the heading
- [ ] And I should see the heading as black (no purple background)
- [ ] And I should see the date and read time in all caps separated by |
- [ ] Did you test in IE 11?

## Visual Regressions

- [x] **Before proceeding:** Run visual regression tests locally to ensure new work does not introduce new bugs.

- [x] _If the new work does your work introduce visual regressions into existing work in the Style Guide (or in a Drupal 8 environment)_, either call these out here or address them before marking the PR as `ready for review`.

- [x] If you are creating or updating a pattern be sure you have created or updated the [scenario in `backstop.json`](ama-style-guide-2/styleguide/backstop.json) and have included a new or updated screenshot in [`bitmaps_reference`](ama-style-guide-2/styleguide/backstop_data/bitmaps_reference).

## Relevant Screenshots/GIFs
screen shot 2018-06-11 at 5 54 04 pm
screen shot 2018-06-11 at 5 53 59 pm

## Remaining Tasks
- [ ] Add to [The Glossary](https://issues.ama-assn.org:8446/confluence/display/DTD/Glossary?src=contextnavpagetreemode)
- [x] Add pattern info
- [x] Add backstop tests

- Not sure of tablet heading size
- Is the whole thing supposed to be a link, or just the text?

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/assets/scss/02-molecules/_category-hero.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    @include breakpoint($bp-small) {
+      background: $purple;
+      padding: 1em .25em 1em 1em;
+
+      a {
+        color: $white;
+        display: block;
+        @include type($myriad-pro, $font-weight-semibold);
+        @include font-size($h1-font-sizes);
```
Comment (labbydev) [inline styleguide/source/assets/scss/02-molecules/_category-hero.scss]: You should be including any `@include` or `@extend` before the other rules.

**Review by labbydev — CHANGES_REQUESTED** [review]
Quick nit picky comment change.

**Comment by labbydev** [comment]
Also...
I'm seeing a number of issues on backstop:
* [ ] Kearning changes on the `ama__search__field`
* [ ] Event page extending the Inline promo incorrectly
* [ ] News as with JAMA page extending the Inline promo incorrectly

screen shot 2018-06-12 at 1 13 34 pm

I believe the extend issue is fixed in PR #282 can you address the kearning changes?

**Comment by csj1012** [comment]
Hi @labbydev - I think the kerning issues with the search text are something @JaeBea is looking at addressing in a separate PR. I've resolved merge conflicts here and made the SCSS change, so this should be ready for review.

**Review by JaeBea — CHANGES_REQUESTED** [review]
Hi @chelsiejohnston,
There are a couple updates needed for this to be merged.

1. The font size for the header and line height is incorrect. Please update to be 38/44.2 as described in the original ticket.

2. Backstop images have not been committed.

**Comment by designerneil** [comment]
@chelsiejohnston I getting a couple of backstop failures.

**Comment by csj1012** [comment]
Hi @designerneil - can you be more specific about the backstop failures?

**Comment by csj1012** [comment]
Hi @JaeBea @designerneil - running `backstop test` on this branch currently, the only backstop failures I'm getting are known issues in `develop` not related to this PR:

- article stub as video - video text overlay (5 failures, one for each breakpoint)
- hub hero as fifty fifty video - video text overlay (5 failures, one for each breakpoint)
- info card as location - Google maps overlay (5 failures, one for each breakpoint)
- hub heroes - video text overlay (3 failure at xs through med breakpoints)
- resource - video text overlay (5 failures, one for each breakpoint)

Suggestion to address these outside of this PR and/or discuss in our Dev Chat meeting tomorrow.

**Review by JaeBea — APPROVED** [review]
Hi @chelsiejohnston,
All requested changes have been addressed. This is good to go!

I will merge into develop.

**Pull request merged**